### DISCUSSION TITLE
Issues with all Claims extending down to Bedrock - even when changing config files

### DISCUSSION THREAD

**Discussion by Katagator** [GriefPrevention/GriefPrevention#1311] [Q&A]
Hi! I've tried looking up the answer to this but can't seem to make any headway.

Using GriefProtection, All claims - admin or players all extend to bedrock. I don't want this to happen, i.e. if someone is strip mining 30 blocks below your base it shouldn't be claimed by the top base owner.

No matter what values I change in the config files about extending and distance below of claims, negative, positive, 0 does it work. It always goes to bedrock. There is probably something I'm missing. Please help!

Running this on a Bukkit floodgate - Geyser-Spigot - Plugins GriefPrevention, LuckPerms, SinglePlayerSleep, HolographicDisplays, Vault, EssentialsX, EssentialsX Chat, and EssentialsXSpawn. 

I assume MaximumDepth: and ExtendIntoGroundDistance: would be the values I need to change? Thanks
I can upload the config file but don't know how to turn it into a file type the website likes!

**Comment by Jikoo** [comment]
To upload your config, it's easiest for us if you just paste it inside a code block (ideally one set to yaml syntax), i.e.
````
```yaml
my:
  config.value: "here"
```
````
results in
```yaml
my:
  config.value: "here"
```

Claims should extend down as players build beneath their limits, so you'll be bound to see claims extending down eventually, but it shouldn't be immediate unless configured to do so.
